### DISCUSSION TITLE
Where can I find the builtin CSS style used for View mode?

### DISCUSSION THREAD

**Discussion by altar729** [gsantner/markor#1488] [Q&A] [answered]
Hello, I have been looking at the source code for a while to find the *CSS style for the View mode*, but I can't find anything.

So what I've been doing is making Markor my story editor and reader.
As I use view mode pretty often, I beautify it by applying codes on Inject -> head in the View mode menu.

The only thing I am not able to change is the display of ```#``` markdown tag. I tried using:

```
h1 {
 color: red;
 font-family: sans;
 border-bottom: 2px solid green;
}
```

yet the only thing got changed is the font family and the border bottom of ```#``` tag content, the color doesn't change.

For example if I write in editor:

```
# This is header using Markdown tag

<h1>This is header using html header tag </h1>
```

The result in the view mode would be:

![image](https://user-images.githubusercontent.com/93818325/140604847-2dd7a803-a499-42a4-8c58-e1aa8946c9ad.png)

Though in the display of ```<h1>``` tag, the text color changes to red, the one on ```#``` didn't.

Would anyone be kind to point me on where to look for to change the color of text with ```#``` markdown tag?

Thank You.

**Comment by gsantner** [comment, marked as answer]
Usually at CSS: If you are sure this should work, and works this way otherwise...an additional `!important` keyword can help at the value definition.

The Markor builtin CSS is differently per syntax/format. What you have to checkout are the TextConverter.java files, i.e. MarkdownTextConverter.  
The easiest way to inspect CSS is to use the Share->HTML(Plaintext) option at editor. This way you can see what view mode renders.

**Reply by altar729** [reply]
Thank you for the hint, It helped me tremendously. 

I had found the solution to my question.

for curious reader from the future, TextConverter.java defines how the plaintext would be translated to the view mode. 
as for the ```H1```, it is rendered using ```H1``` which text is then converted to render using ```a:link{ }```

so the style header should be:
```
h1 {
 font-family: sans;
 border-bottom: 2px solid green;
}

a:link {
 color: red;
}
```
